src/Controller/RegistrationController.php

```diff
@@ -90,16 +95,24 @@ public function registration(Request $request): Response
      *
      * @throws \InvalidArgumentException
      */
+    #[RequiresActiveSession]
     #[Route(path: '/registration/status', name: 'app_identity_registration_status', methods: ['GET'])]
     public function registrationStatus() : Response
     {
         $this->logger->info('Request for registration status');
-        // Do have a valid sample AuthnRequest?.
+
+        // Do we have a valid GSSP registration AuthnRequest in this session?
         if (!$this->registrationService->registrationRequired()) {
             $this->logger->error('There is no pending registration request');
 
             return new Response('No registration required', Response::HTTP_BAD_REQUEST);
         }
+
+        if ($this->tiqrService->isEnrollmentTimedOut()) {
```

Comment:
Prior to this change, I assume the timeouts would occur, but the tiqr 'frontend' would be unaware?